app/policies/admin/claim_policy.rb

```diff
@@ -4,18 +4,22 @@ class Scope < Struct.new(:user, :scope)
       def resolve
         scope
       end
-
     end
+
     def index?
-      true
+      user.is_admin? || user.permission_names.include?('read_claims')
```

Comment:
did you thought about refactoring the user.permission_names.include?('read_claims') into something like use.has_permission?('reading_claims')